plonetheme/onegov/patterns/configure.zcml

```diff
@@ -0,0 +1,9 @@
+<configure xmlns="http://namespaces.zope.org/zope">
+
+  <adapter
+      for="* * *"
```

Comment:
What do three stars do? I only know one for "everything" yet.

Reply:
This is needed because the implementation [searches for adapters with 3 given interfaces](https://github.com/plone/Products.CMFPlone/blob/fc45ebf97b5f10aa690e405c9acd5cfde65a5616/Products/CMFPlone/patterns/view.py#L36-L39) (self.context, self.request, None). In our adapter the third argument is `field` because this is how it is done here: https://github.com/plone/Products.CMFPlone/blob/fc45ebf97b5f10aa690e405c9acd5cfde65a5616/Products/CMFPlone/patterns/settings.py#L30-L33